src/components/Footer/RenderCompanyInfoSection.jsx

```diff
@@ -5,9 +5,20 @@ import Box from '@mui/material/Box';
 import Link from '@mui/material/Link';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
-import TwitterIcon from '@mui/icons-material/Twitter';
+
 import FacebookIcon from '@mui/icons-material/Facebook';
 import InstagramIcon from '@mui/icons-material/Instagram';
+import SvgIcon from '@mui/material/SvgIcon';
+
+// new Twitter Icon
+const TwitterIcon = (props) => (
```

Comment:
If this is simply a new custom component for the new Twitter Icon, I don't think we need to add a prop to this.